benchmarks/benchmark_serving.py

```diff
@@ -685,7 +761,7 @@ def main(args: argparse.Namespace):
         "--dataset-name",
         type=str,
         default="sharegpt",
-        choices=["sharegpt", "sonnet", "random"],
+        choices=["sharegpt", "sonnet", "random", "hf"],
         help="Name of the dataset to benchmark on.",
```

Comment:
Can you update the `help` for `--dataset-path`, in particular to call out that huggingface dataset ID can be specified for `hf`?